smart-contracts/osmosis/contracts/cl-vault/src/contract.rs

```diff
@@ -214,8 +216,27 @@ pub fn reply(deps: DepsMut, env: Env, msg: Reply) -> Result<Response, ContractEr
 }
 
 #[cfg_attr(not(feature = "library"), entry_point)]
-pub fn migrate(deps: DepsMut, _env: Env, _msg: MigrateMsg) -> Result<Response, ContractError> {
-    MIGRATION_STATUS.save(deps.storage, &MigrationStatus::Open)?;
+pub fn migrate(deps: DepsMut, _env: Env, msg: MigrateMsg) -> Result<Response, ContractError> {
```

Comment:
Add a unit test for this behaviour in the migration please, this also helps in clarifying for the next rollout what you expect to happen here

Reply:
Done!